roles/smallfile-bench/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+---
+- set_fact:
+    smallfile_path: /mnt/pvc
+  when: smallfile.storageclass is defined
+
+- set_fact:
+    smallfile_path: /tmp/test_smallfile/
+  when: smallfile.storageclass is not defined
```

Comment:
why is it necessary to support the case where no storage class is defined?   If we just always required a storage class this would be simpler.   Minikube offers local storage class so there should not be a problem even when there is no Ceph.

Reply:
Ok, then, there is no problem to remove this.